src/main/java/uk/gov/hmcts/reform/sscs/service/RoboticsService.java

```diff
@@ -41,33 +46,53 @@ public RoboticsService(
     }
 
     public void sendCaseToRobotics(SscsCaseData caseData, Long caseId, String postcode, byte[] pdf) {
+        sendCaseToRobotics(caseData, caseId, postcode, pdf, Collections.emptyMap());
+    }
+
+    public void sendCaseToRobotics(SscsCaseData caseData, Long caseId, String postcode, byte[] pdf, Map<String, byte[]> additionalEvidence) {
         String venue = airLookupService.lookupAirVenueNameByPostCode(postcode);
 
         JSONObject roboticsJson = createRobotics(RoboticsWrapper.builder().sscsCaseData(caseData)
                 .ccdCaseId(caseId).venueName(venue).evidencePresent(caseData.getEvidencePresent()).build());
 
-        sendJsonByEmail(caseData.getAppeal().getAppellant(), roboticsJson, pdf);
+        sendJsonByEmail(caseData.getAppeal().getAppellant(), roboticsJson, pdf, additionalEvidence);
         log.info("Robotics email sent successfully for Nino - {} and benefit type {}", caseData.getAppeal().getAppellant().getIdentity().getNino(),
                 caseData.getAppeal().getBenefitType().getCode());
     }
 
     public JSONObject createRobotics(RoboticsWrapper appeal) {
 
         JSONObject roboticsAppeal =
-            roboticsJsonMapper.map(appeal);
+                roboticsJsonMapper.map(appeal);
 
         roboticsJsonValidator.validate(roboticsAppeal);
 
         return roboticsAppeal;
     }
 
-    private void sendJsonByEmail(Appellant appellant, JSONObject json, byte[] pdf) {
+    private void sendJsonByEmail(Appellant appellant, JSONObject json, byte[] pdf, Map<String, byte[]> additionalEvidence) {
         String appellantUniqueId = emailService.generateUniqueEmailId(appellant);
-        emailService.sendEmail(roboticsEmailTemplate.generateEmail(
-                appellantUniqueId,
-                newArrayList(json(json.toString().getBytes(), appellantUniqueId + ".txt"),
-                        pdf(pdf, appellantUniqueId + ".pdf"))
-        ));
+        List<EmailAttachment> attachments = addDefaultAttachment(json, pdf, appellantUniqueId);
+        addAdditionEvidenceAttachments(additionalEvidence, attachments);
+        emailService.sendEmail(
+                roboticsEmailTemplate.generateEmail(
+                        appellantUniqueId,
+                        attachments
+                )
+        );
+    }
+
+    private void addAdditionEvidenceAttachments(Map<String, byte[]> additionalEvidence, List<EmailAttachment> attachments) {
```

Comment:
typo in the method name 'Addition' should be 'Additional'

Reply:
done